Index.html

```diff
@@ -28,6 +28,204 @@ <h2 id="myTitle">Software Developer</h2>
+                <a id="languages" href="#"><h2>Languages</h2></a>
+                <ul class="abilities">
+                    <li>
+                        <a href="#">JavaScript</a>
+                    </li>
+                    <li>
+                        <a href="#">Ruby</a>
+                    </li>
+                    <li>
+                        <a href="#">Html</a>
+                    </li>
+                    <li>
+                        <a href="#">Html</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="mySkills">
+                <a id="frameworks" href="#"><h2>Frameworks</h2></a>
+                <ul class="abilities">
+                    <li>
+                        <a href="#">Bootstrap</a>
+                    </li>
+                    <li>
+                        <a href="#">Ruby on Rails</a>
+                    </li>
+                    <li>
+                        <a href="#">RSPec</a>
+                    </li>
+                    <li>
+                        <a href="#">CapyBara</a>
+                    </li>
+                    <li>
+                        <a href="#">Selenium</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="mySkills">
+                <a id="skills" href="#"><h2>Skills</h2></a>
+                <ul class="abilities">
+                    <li>
+                        <a href="#">Codekit</a>
+                    </li>
+                    <li>
+                        <a href="#">Github</a>
+                    </li>
+                    <li>
+                        <a href="#">Codepen</a>
+                    </li>
+                    <li>
+                        <a href="#">Gitlab</a>
+                    </li>
+                    <li>
+                        <a href="#">Terminal</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <img class="line" src="./assets/vectorLine.svg" alt="horizontal line">
+    </section>
+    <section class="socials">
+        <ul class="profiles">
+            <li>
+                <a href="#">
+                    <img id="github" src="./assets/github.svg" alt="GitHub Logo">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img id="linkedIn" src="/assets/linkedinn.png" alt="LinkedIn Logo">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img id="angellist" src="/assets/angellist.svg" alt="Angellist Logo">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img id="twitter" src="/assets/Twitter.svg" alt="Twitter Logo">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img id="medium" src="/assets/medium.svg" alt="Medium Logo">
+                </a>
+            </li>
```

Comment:
- [ ] Very great job while implementing this list of social media, just there is something that we need to improve here.
   - It seems that the icons are displaying on your side but it's not the case on my end, look how it looks like:
      - ![iconb2](https://user-images.githubusercontent.com/57562869/143579932-ae7d4754-51b7-4c3b-bfe8-9523215b31d7.png)
      - This is the problem of the path of images. it seems that you did not put the right path, it doesn't work on my end because Github has case sensitive paths, it's supposed to be very correct.
      - Kindly try to fix that, by fixing that, it will improve the look of your portofolio. :+1: